README.md

```diff
@@ -114,50 +114,43 @@ Set up a `SignInWithAppleButton` via XML:
     app:sign_in_with_apple_button_cornerRadius="4dp" />
 ```
 
-In your Activity, create the `SignInWithAppleService`, implement `SignInWithAppleClient`, and configure the button with both:
+In your Activity, create the `SignInWithAppleService`, and configure the button with it:
```

Comment:
Above this change, we explain `SignInWithAppleClient`, which we can replace with an explanation of the callback and, separately, the purpose of supplying the `FragmentManager`.